### PR TITLE
Fixes lottery taking into account departments, and wild html tags

### LINKED ISSUE #2832: BUG: Lottery
[Common] CEV Eris System Announcer states, "Lottery: Nyx Daily wishes to congratulate `<b></b>` for winning the Nyx Stellar Slam Lottery, and receiving the out of this world sum of 15000 credits!"

### PR DESCRIPTION
**Pull request by Carbonhell**
Fixes #2832 and removes the chance for departments to be chosen as winners.

### CODE REVIEW

File: code/game/gamemodes/events/money_lotto.dm [hunk 1]
```diff
@@ -24,8 +24,11 @@
 /datum/event/money_lotto/start()
 	winner_sum = pick(5000, 10000, 15000)
 	if(all_money_accounts.len)
-		var/datum/money_account/D = pick(all_money_accounts)
-		winner_name = D.owner_name
+		var/list/private_accounts = all_money_accounts.Copy()
+		for(var/i in department_accounts)
```
Comment (NanakoAC) [inline code/game/gamemodes/events/money_lotto.dm]: this is unnecessary, you can just directly subtract a whole list, like this

private_accounts -= department_accounts

File: html/changelogs/lottery.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+author: Carbonhell
+delete-after: True
+changes: 
+  - bugfix: "The lottery cannot be win by departments anymore, and the output message has been fixed."
```
Comment (mattroks101) [inline html/changelogs/lottery.yml]: Be won.

File: code/game/gamemodes/events/money_lotto.dm (hunk 1 above)
Comment (Carbonhell) [reply code/game/gamemodes/events/money_lotto.dm]: Department accounts is a list with numbers as keys and the actual accounts as value, i thought simply doing -= would attempt to subtract the keys, huh

Comment (NanakoAC) [reply code/game/gamemodes/events/money_lotto.dm]: oh i see, disregard this comment then

**Pull request merged**